.github/actions/cache/action.yml

```diff
@@ -25,6 +28,16 @@ runs:
         echo "Current workflow path is: ${MIDDLE}"
         echo "path=${MIDDLE}" >> "$GITHUB_OUTPUT"
       shell: bash
+    - name: Generate meson cache key
+      if: ${{ inputs.use-meson == 'true' }}
+      run: |
+        echo "MESON_KEY=`meson --version`" >> $GITHUB_ENV
```

Comment:
I wonder if we should include `meson-` in the value of `MESON_KEY` to make it more reasable:

```
        echo "MESON_KEY=meson-`meson --version`" >> $GITHUB_ENV
```

Reply:
Agreed.